src/Symfony/Component/Translation/Remote/Dsn.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Translation\Remote;
+
+use Symfony\Component\Translation\Exception\InvalidArgumentException;
+
+/**
+ * @author Fabien Potencier <fabien@symfony.com>
+ *
+ * @experimental in 5.1
+ */
+final class Dsn
```

Comment:
So another class for a DSN... Wasn't there some discussions on that point ?

Reply:
Yes, @Nyholm has opened a PR about a new DSN component (https://github.com/symfony/symfony/pull/37462)